docs/api_documentation.md

```diff
@@ -0,0 +1,45 @@
+# API Docs
+## Generating Docs
+Docs are autogenerated based off of the requests specs, no extra code is needed.
+
+The specs are run with the OPENAPI env var set to trigger the doc generation, it is set on pushes to main and when changes are made to `spec/requests/api`.
+An autolabeler action takes care of labeling for any PR that changes the `spec/requests/api` files.
+
+*The label has to be created before hand for the labeler to work*
```

Comment:
no longer necessary I think?

Reply:
we still do it, maybe this is not the place for it. CI might be better since it's more generic now not necessarily tied to the docs (even though is the only current label)